jwst/residual_fringe/utils.py

```diff
@@ -840,3 +841,408 @@ def new_make_knots(flux, nknots=20, weights=None):
         knot_idx = np.unique(knot_idx.astype(int))
 
     return knot_idx.astype(int)
+
+# RFC1D additions ======================================
+
+# constants
+FFREQ_1d = [2.9, 0.4]
+DFFREQ_1d = [1.5, 0.15]
+MAX_NFRINGES_1d = [10, 15]
+MAXAMP_1d = 0.2
```

Comment:
They could be, but I'd prefer to leave that to a future improvement if necessary.  All the infrastructure associated with new ref files is unnecessary at the moment since this is an optional non-default way of running the pipeline anyway.

First value is main fringe frequency, second is dichroic fringe frequency only relevant to ch 3+4.

Added some comments.